<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkAblationPlanningViewControls</class>
 <widget class="QWidget" name="QmitkAblationPlanningViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>864</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoSelectSegmentationImageLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Selected segmentation:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkDataStorageComboBox" name="segmentationComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="safetyMarginLabel">
       <property name="text">
        <string>Safety Margin (mm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="safetyMarginSpinBox">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calculateSafetyMarginPushButton">
     <property name="text">
      <string>Calculate Safety Margin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ablationStartingPointPushButton">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Choose Ablation Starting Point</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ablationStartingPointLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Segmentation Starting Point: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ablationRadiusLabel">
       <property name="text">
        <string>Ablation Radius (mm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ablationRadiusSpinBox">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>60.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="tissueShrinkingLabel">
       <property name="text">
        <string>Tissue Shrinking (in %):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tissueShrinkingSpinBox">
       <property name="maximum">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="gridModelRadioButton">
       <property name="text">
        <string>Use Grid Model</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="randomDistributionRadioButton">
       <property name="text">
        <string>Use Random Distribution Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="repitionsCalculatingAblationZonesLabel">
       <property name="text">
        <string>Repitions Calculating Ablation Zone Distribution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="repititionsCalculatingAblationZonesSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="toleranceNonAblatedTumorSafetyMarginVolumeLabel">
       <property name="text">
        <string>Tolerance Non-ablated Volume (in %):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toleranceNonAblatedTumorSafetyMarginVolumeSpinBox">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="refreshCalculationsPushButton">
     <property name="text">
      <string>Values Changed - A Refresh Of Calculations Might Be Needed</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../../src/Modules/USUI/resources/USUI.qrc">
       <normaloff>:/USUI/restart.png</normaloff>:/USUI/restart.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculateAblationZonesPushButton">
     <property name="text">
      <string>Calculate Ablation Zones</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../../src/Modules/USUI/resources/USUI.qrc">
       <normaloff>:/USUI/accept.png</normaloff>:/USUI/accept.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="ablationVoluminaLabel">
       <property name="text">
        <string>Number Ablation Volumina:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="numberAblationVoluminaLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="numberTumorVolumeLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tumorVolumeLabel">
       <property name="text">
        <string>Tumor Volume (in ml):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="tumorAndMarginVolumeLabel">
       <property name="text">
        <string>Tumor Volume + Safety Margin (in ml):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="numberTumorAndMarginVolumeLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ablationVolumeLabel">
       <property name="text">
        <string>Total Ablation Volume (in ml):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="numberAblationVolumeLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="volumeAblatedTwoAndMoreLabel">
       <property name="text">
        <string>Volume ablated 2-/3-/4-times (in ml):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="numberVolumeAblatedTwoAndMoreLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="overlappingAblationZonesLabel">
       <property name="text">
        <string>Factor Overlapping Of Ablation Zones (in %):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="numberOverlappingAblationZonesLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="factorAblatedVolumeOutsideSafetyMarginLabel">
       <property name="text">
        <string>Factor Ablated Volume Outside Safety Margin (in %)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="numberFactorAblatedVolumeOutsideSafetyMarginLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="editAblationZones">
     <property name="title">
      <string>Edit Ablation Zones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="descriptiveLabelEditingAblationZones">
        <property name="text">
         <string>For changing the position of a given ablation zone, please:

 1) Choose the ablation zone you want to move within the list below.
 2) Mark the new center of the chosen ablation zone within the segmentation image by your mouse.
 3) Confirm the new position by clicking the button &quot;Confirm New Position&quot;.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ablationZonesComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="confirmNewPositionPushButton">
        <property name="text">
         <string>Confirm New Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>For deleting an ablation zone, please:

1) Choose the ablation zone you want to remove within the list above.
2) Click the button &quot;Delete Chosen Ablation Zone&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteChosenAblationZonePushButton">
        <property name="text">
         <string>Delete Chosen Ablation Zone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>For adding a new ablation zone, please:

 1) Mark the center of the new ablation zone within the segmentation image by your mouse.
 2) Confirm the new ablation zone by clicking the button &quot;Add New Ablation Zone&quot;.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNewAblationZonePushButton">
        <property name="text">
         <string>Add New Ablation Zone</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../src/Modules/USUI/resources/USUI.qrc"/>
 </resources>
 <connections/>
</ui>
